plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/ILibraryClasspathContainerResolverService.java

```diff
@@ -30,8 +30,27 @@
  */
 public interface ILibraryClasspathContainerResolverService {
 
+  // extract to a standalone enum?
```

Comment:
yes please.
That sais,

1. I'm surprised this is in the libraries package. It doesn't seem to belong there.
2. How will this work with runtimes for flex?